CIP-1776/README.md

```diff
@@ -0,0 +1,84 @@
+
+4. Perverse incentives: The proposal as put forward by IOG entrenches existing powers rather than truly grant control over the Cardano Blockchain to the community. 
+
+5. Exclusionary Representation: Current discussions fo CIP 1694 effectively exclude members of the developing word from access by introducing a threshold to become a DRep. 
+
+The author proposes a new alternative, a non-incentivized iterative block approval voting system to manage treasury disbursement (that would be a variant of multi-winner approval voting modified to resolve the Burr dilemma) and a three-house structure for overarching governance actions. This alternative would be more decentralized, and the ratification of governance actions would be the responsibility of SPOs, Dreps, and a highly modified constitutional committee herein the ‘SC’ or Supreme Court. Let us apply Occam's Razor, put simply “the simplest solution is almost always the best". A system of arbitrary complexity is often developed those of great intelligence - this issue plagues particle physics where increasingly complex models are proposed as a refutation to the standard model, requiring increasingly complex experimentation which ultimately results in increasingly expensive refutation of the underlying theory. This problem manifests in almost all fields of academia. 
+
+Further, the author proposes an alternative form of Cardano governance, radically improving upon the experimental Project Catalyst which in effect resolves systematic issues uncovered during the priciple experiment while incoperating all treasury actions as a formalized governance action. 
+
+----
+##A series of bullet points / template
+
+- System can be divided simply into three houses
+    - House of Dreps 
+    - Congress of SPOs 
+    - Supreme Court
+        (for the love of god calling it the contitutional comittee is utterly foolish. Do we want the same central planning comittee of the failed communist states? Ridiculous!)
+        
+HOUSE;
+Anyone can become a drep, and therefore anyone can take part in the Governance of Cardano. The absurd suggestion of CIP 1694 that their should be a financial threshold for participation in governance ultimately disenfranchises those from those in the developing world. The ability to get support for a governance action should ABSOLUTELY NOT be exclusionary. The House of Dreps, or simply The House, is to be considered the lower chamber of governance due to the lower barrier to entry. Anyone can become a member of the house, and by default independent voters are by virtue of holding ADA (and a 1 ADA registration fee) are members of the House. The house would naturally become a chamber of many, many proposals - more proposals that can feasably be considered, thus the tripartate system requireing ratification by the congress and acent by the Supreme Court. 
```

Comment:
I mentioned this above, but just to put a finer point on it: isn't a 1 ada registration fee a "financial threshold" that will "ultimately disenfranchise" voters?

Reply:
That's correct, drafts of v1 call for only a transaction fee.